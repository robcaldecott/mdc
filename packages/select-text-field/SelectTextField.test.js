import React from "react";
import { SelectTextField } from ".";
import renderer from "react-test-renderer";

it("renders", () => {
  expect(
    renderer.create(
      <SelectTextField label="Pick a food group">
        <option value="grains">Bread, Cereal, Rice, and Pasta</option>
        <option value="vegetables" disabled>
          Vegetables
        </option>
        <option value="fruit">Fruit</option>
        <option value="dairy">Milk, Yogurt, and Cheese</option>
        <option value="meat">
          Meat, Poultry, Fish, Dry Beans, Eggs, and Nuts
        </option>
        <option value="fats">Fats, Oils, and Sweets</option>
      </SelectTextField>
    )
  ).toMatchSnapshot();
});
