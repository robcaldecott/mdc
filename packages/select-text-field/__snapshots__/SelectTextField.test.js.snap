// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<label
  className="mdc-text-field mdc-text-field--upgraded"
  style={
    Object {
      "backgroundImage": "url(data:image/svg+xml,%3Csvg%20width%3D%2210px%22%20height%3D%225px%22%20viewBox%3D%227%2010%2010%205%22%20version%3D%221.1%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20xmlns%3Axlink%3D%22http%3A%2F%2Fwww.w3.org%2F1999%2Fxlink%22%3E%0A%20%20%20%20%3Cpolygon%20id%3D%22Shape%22%20stroke%3D%22none%22%20fill%3D%22%230%22%20fill-rule%3D%22evenodd%22%20opacity%3D%220.54%22%20points%3D%227%2010%2012%2015%2017%2010%22%3E%3C%2Fpolygon%3E%0A%3C%2Fsvg%3E)",
      "backgroundPosition": "right 4px bottom 16px",
      "backgroundRepeat": "no-repeat",
    }
  }
>
  <select
    className="mdc-text-field__input"
    disabled={undefined}
    onBlur={[Function]}
    onFocus={[Function]}
    style={
      Object {
        "paddingRight": 20,
      }
    }
    value={undefined}
  >
    <option
      value=""
    />
    <option
      value="grains"
    >
      Bread, Cereal, Rice, and Pasta
    </option>
    <option
      disabled={true}
      value="vegetables"
    >
      Vegetables
    </option>
    <option
      value="fruit"
    >
      Fruit
    </option>
    <option
      value="dairy"
    >
      Milk, Yogurt, and Cheese
    </option>
    <option
      value="meat"
    >
      Meat, Poultry, Fish, Dry Beans, Eggs, and Nuts
    </option>
    <option
      value="fats"
    >
      Fats, Oils, and Sweets
    </option>
  </select>
  <span
    className="mdc-text-field__label"
  >
    Pick a food group
  </span>
  <div
    className="mdc-text-field__bottom-line"
  />
</label>
`;
